styleguide/derek/global-components/tracking/tracking.js

```diff
@@ -181,6 +181,35 @@ Zoolander.Tracking = (function Tracking($) {
         eventNonInteraction: 0,
       });
     });
+
+    $('form.rsFilter-form').on('submit', (e) => {
+      dataLayer.push({
+        event: 'ga.event',
+        eventCategory: 'Resource Center',
+        eventAction: 'Resource Center Filter - Product',
+        eventLabel: $(e.target).find('option:selected').text(),
+        eventValue: '0',
+        eventNonInteraction: 0,
+      });
+
+      dataLayer.push({
+        event: 'ga.event',
+        eventCategory: 'Resource Center',
+        eventAction: 'Resource Center Filter - Type',
+        eventLabel: getCheckboxes($(e.target).find('input:checked')),
+        eventValue: '0',
+        eventNonInteraction: 0,
+      });
+    });
+  }
+
+  function getCheckboxes(elements) {
```

Comment:
Can we rename this to something a little bit more specific? Something like `getCheckboxValuesByElement()`.  It is a bit more verbose, but it better explains what the function is doing.